nano/templates/marine_selector.tmpl

```diff
@@ -1,35 +1,135 @@
+<style>
```

Comment:
are the PR description screenshot(s) this template?

Reply:
Yeah

Reply:
looks good